test/cli-test.js

```diff
@@ -7,13 +7,31 @@ var cliTxt = path.resolve(_files, 'cli.txt');
 var cliPartialsTxt = path.resolve(_files, 'cli_with_partials.txt');
 var moduleVersion = require('../package').version;
 
-var exec = require('child_process').exec;
+var child_process = require('child_process');
+
+function changeForOS(command) {
+
+  if(process.platform === 'win32') {
+    return command.
+      replace(/bin\/mustache/g, 'node bin\\mustache').
+      replace(/\bcat\b/g, 'type').
+      replace(/\//g, '\\');
```

Comment:
When chaining, dots are usually put in the beginning of the line:
```js
if (...) {
  return command
  .replace(...)
  .replace(...)
  .replace(...)
}
```
Also, I'd say the blank lines inside the function are unnecessary. Specially the first one. Blank lines are mainly used to separate chunks / logic sections, and there's not much to separate here.